File: test/net/dinkla/raytracer/samplers/JitteredTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class JitteredTest extends GeneratorTest {

    @BeforeEach
    @Override
    public void initialize() {
        new Jittered().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }

}

File: src/test/kotlin/net/dinkla/raytracer/samplers/JitteredTest.kt
package net.dinkla.raytracer.samplers

import org.junit.jupiter.api.BeforeEach

class JitteredTest : AbstractGeneratorTest() {

    @BeforeEach
    override fun initialize() {
        Jittered().generateSamples(NUM_SAMPLES, NUM_SETS, samples)
    }

}
